sync/src/relayer/compact_block_process.rs

```diff
@@ -85,6 +85,19 @@ impl<'a> CompactBlockProcess<'a> {
 
         let status = snapshot.get_block_status(&block_hash);
         if status.contains(BlockStatus::BLOCK_STORED) {
+            // update last common header and best known
+            let parent = snapshot
+                .get_header_view(&header.data().raw().parent_hash())
+                .unwrap();
```

Comment:
Better use `expect`. I am afraid that [snapshot problem](https://github.com/nervosnetwork/ckb/pull/1947) may cause here panic.

Reply:
No, when the block exists locally, its parent block must exist. If it does not exist, it is a very serious bug.

Reply:
Anyway, `expect` is better than `unwrap`.

Reply:
Does `BLOCK_STORED` guarantee that all the ancestors are also stored?

Reply:
> Does `BLOCK_STORED` guarantee that all the ancestors are also stored?

yes